ocs_ci/helpers/e2e_helpers.py

```
@@ -113,6 +153,1046 @@ def get_all_combinations_map(providers, bucket_types):
+                upload_id,
+                bucket_name,
+            )
+            return upload_id
+
+        logger.warning(
+            "Multipart upload %s has not been aborted yet on %s: %s",
+            upload_id,
+            bucket_name,
+            uploads,
+        )
+    else:
```

Ruff (0.15.17)

[warning] `else` clause on loop without a `break` statement; remove the `else` and dedent its contents

Remove `else`

(PLW0120)